pkg/scheduler/framework/plugins/defaultpreemption/default_preemption_test.go

```diff
@@ -52,10 +52,17 @@ import (
 	frameworkruntime "k8s.io/kubernetes/pkg/scheduler/framework/runtime"
 	framework "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
 	internalcache "k8s.io/kubernetes/pkg/scheduler/internal/cache"
+	"k8s.io/kubernetes/pkg/scheduler/internal/parallelize"
 	internalqueue "k8s.io/kubernetes/pkg/scheduler/internal/queue"
 	st "k8s.io/kubernetes/pkg/scheduler/testing"
 )
 
+func init() {
+	// set the default parallelize.Parallelism to 16 as scheduler defaults are not
+	// loaded in with generic scheduler
+	parallelize.Parallelism = 16
+}
+
```

Comment:
I decided to add this in the `init` as there was no simpler way to configure this for all tests, some rely on `genericScheduler` and other directly invoke the preemption and other business logics, this way have a common solution for all tests in a package, or we can do this at a per test level

Reply:
Please remove this init. See comment below.